recipes-kernel/linux/linux-iot2050-6.1.inc

```diff
@@ -26,6 +26,8 @@ def get_extra_configs(d):
 
     if re.search("pvu-dma", overrides):
         append_config.append('CONFIG_TI_PVU=y')
+        if re.search("meta-hailo", overrides):
+            append_config.append('CONFIG_SWIOTLB_SEGSIZE=256')
```

Comment:
Will go in that config snippet that the previous patch should introduce.

Reply:
That is no problem because then the default value will remain in place - you are bending the kernel unconditionally.

Reply:
ACK